### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by enahum**
@saturninoabril take a look so you can rebase your PR #1637

### CODE REVIEW

File: app/screens/image_preview/downloader.ios.js [hunk 1]
```diff
@@ -183,18 +156,58 @@ export default class Downloader extends PureComponent {
...
+                {component}
             </View>
+
         );
     };
 
     renderStartDownload = () => {
         return (
-            <View style={styles.progressContent}>
+            <View>
```
Comment (hmhealey) [inline app/screens/image_preview/downloader.ios.js]: Does this View do anything any more, or can it be removed?

File: package.json [hunk 2]
```diff
@@ -15,47 +15,47 @@
...
     "intl": "1.2.5",
-    "jail-monkey": "0.2.0",
+    "jail-monkey": "1.0.0",
     "mattermost-redux": "mattermost/mattermost-redux#c2cdd8fad33b97af4d0af2462b24f9cfd50a1913",
     "prop-types": "15.6.1",
-    "react": "16.3.1",
+    "react": "16.3.2",
     "react-intl": "2.4.0",
-    "react-native": "0.52.3",
+    "react-native": "0.55.3",
```
Comment (hmhealey) [inline package.json]: RN 55, round 2

File: app/screens/image_preview/downloader.ios.js (hunk 1 above)
Comment (enahum) [reply app/screens/image_preview/downloader.ios.js]: Without this View it does not show

File: package.json [hunk 3]
```diff
@@ -15,47 +15,47 @@
...
     "react-native-tooltip": "5.2.0",
-    "react-native-vector-icons": "4.5.0",
+    "react-native-vector-icons": "4.6.0",
     "react-native-video": "2.0.0",
     "react-native-youtube": "1.1.0",
-    "react-navigation": "1.5.8",
+    "react-navigation": "1.5.11",
     "react-redux": "5.0.7",
-    "redux": "3.7.2",
+    "redux": "4.0.0",
```
Comment (saturninoabril) [inline package.json]: Should we do this first with `mattermost-redux`?

Comment (enahum) [reply package.json]: https://pre-release.mattermost.com/core/pl/3d9gewmjg3y9dei7u1djdzqxac

**Pull request merged**